js_modules/dagit/packages/app/src/NUX/CommunityNux.tsx

```diff
@@ -0,0 +1,145 @@
+import {useStateWithStorage} from '@dagster-io/dagit-core/hooks/useStateWithStorage';
+import {
+  Body,
+  Box,
+  Button,
+  Checkbox,
+  Colors,
+  Dialog,
+  ExternalAnchorButton,
+  Heading,
+  Icon,
+  TextInput,
+} from '@dagster-io/ui';
+import React from 'react';
+import isEmail from 'validator/lib/isEmail';
+
+export const CommunityNux = () => {
+  const [didDismissCommunityNux, dismissCommunityNux] = useStateWithStorage(
+    'communityNux',
+    (data) => data,
+  );
+  if (didDismissCommunityNux) {
+    return null;
+  }
+  return (
+    <CommunityNuxImpl
+      dismiss={() => {
+        dismissCommunityNux('1');
+      }}
+    />
+  );
+};
+
+const CommunityNuxImpl: React.FC<{dismiss: () => void}> = ({dismiss}) => {
+  const [shouldShowNux, setShouldShowNux] = React.useState(false);
+  React.useEffect(() => {
+    const timeout = setTimeout(() => {
+      setShouldShowNux(true);
+    }, 5000); // wait 5 seconds before showing Nux
```

Comment:
Five seconds seems almost too long, like it might be jarring to do stuff for a few seconds and only then see the dialog.

Reply:
Josh suggested 5 seconds. I think it's reasonable, though I do agree it can be jarring. The good news is that its not like a full page dialog or anything so it shouldn't interfere with you doing something